### PR TITLE
Lab2

### PR DESCRIPTION
**Pull request by Homa3030**

### CODE REVIEW

**Comment by afonyaa** [comment]
Good work! The dockerfile meets practices, pointed in .md  file

File: app_python/.gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+__pycache__/*
```
Comment (AKurmazov) [inline app_python/.gitignore:1]: fyi: There is a [cool resource](https://www.toptal.com/developers/gitignore/) that generates a .gitignore file based on the technologies you use

File: app_python/DOCKER.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# About Dockerfile
+## Dockerfile Best Practices
+
+- Exposed ports
```
Comment (AKurmazov) [inline app_python/DOCKER.md:4]: Not a good practice in general :)

File: app_python/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# syntax=docker/dockerfile:1
+
+FROM python:3.8-slim-buster
+WORKDIR /app
+COPY requirements.txt requirements.txt
+RUN pip3 install -r requirements.txt
+COPY README.md app.py PYTHON.md requirements.txt ./
```
Comment (AKurmazov) [inline app_python/Dockerfile]: You probably do not need your MD files in the container. And also the requirements file is copied twice

**Comment by Maxkoz777** [comment]
Everything is ok!

**Comment by none-word** [comment]
Good job!

**Pull request closed without merging**